openlibrary/catalog/update_count.py

```diff
@@ -6,7 +6,7 @@
 
 def commify(n):
     """
-Add commas to an integer `n`.
+Add commas to an integer repr(n).
```

Comment:
@salman-bhai , @mekarpeles I see this has been merged, but repr() isn't appropriate here in a docstring, should just be normal single quotes

Reply:
My preferences, in order, are n, N, 'n', "n". I don't think the quotes are needed for a single character variable name.

Reply:
I'll change this in a separate PR! Will go through this again and get it done!